core/misc/textarea.js

```diff
@@ -2,30 +2,8 @@
 
 Backdrop.behaviors.textarea = {
   attach: function (context, settings) {
```

Comment:
I think it'd be pretty safe to go ahead and remove this entire file.

Reply:
Then, let's do it.